morango/controller.py

```diff
@@ -62,6 +72,10 @@ def serialize_into_store(self):
                             'profile': app_model.morango_profile,
                             'partition': app_model._morango_partition,
                         }
+                        (self_ref_fk_bool, self_ref_fk_name) = _self_referential_fk(klass_model)
+                        if self_ref_fk_bool:
+                            self_ref_fk_value = getattr(app_model, self_ref_fk_name)
+                            kwargs.update({'_self_ref_fk': self_ref_fk_value if self_ref_fk_value else ''})
```

Comment:
`self_ref_fk_value if self_ref_fk_value else ''` could just be `self_ref_fk_value or ''`